testsuite/features/support/constants.rb

```diff
@@ -142,7 +142,7 @@
   end
 
 proxy_base_channel =
-  if $product == 'Uyuni'
+  if ENV['SERVER'].include? 'uyuni'
```

Comment:
```suggestion
  if product == 'Uyuni'
```

Reply:
This does not work.

```bash
uyuni-bv-master-ctl:~/spacewalk/testsuite # rake cucumber:build_validation_add_activation_key_proxy
/usr/bin/ruby.ruby2.5 -S bundle exec cucumber --profile default -f html -o results/output_20230808114149-build_validation_add_activation_key_proxy.html -f json -o results/output_20230808114149-build_validation_add_activation_key_proxy.json -f junit -o results_junit -f pretty -r features features/build_validation/add_activation_keys/proxy_add_activation_key.feature
Capybara APP Host: https://uyuni-bv-master-srv.mgr.suse.de:8888
undefined method `run' for nil:NilClass (NoMethodError)
/root/spacewalk/testsuite/features/support/commonlib.rb:54:in `product'
/root/spacewalk/testsuite/features/support/constants.rb:138:in `<top (required)>'
```

Reply:
:S Ok, I see, this happen to early, ok, lgtm them.